pandas/core/generic.py

```diff
@@ -2620,7 +2620,7 @@ def to_hdf(
             - 'a': append, an existing file is opened for reading and
               writing, and if the file does not exist it is created.
             - 'r+': similar to 'a', but the file must already exist.
-        complevel : {0-9}, optional
+        complevel : {0-9}, default None
```

Comment:
if the user doesn't pass anything, does it end up not compressing, i.e. equivalent to complevel=0?

Reply:
In HDFStore it reads:  `A value of 0 or None disables compression.`

Should replace the second sentence here with that one.

Reply:
Hello @bashtage , `A value of 0 or None disables compression.` instead of `default None` this is the suggestion right ?

Reply:
This should be added the the explanation.  See the explanation on HDFStore.